### PR TITLE
feat: Upload csv file

### PR DESCRIPTION
**Pull request by kelvin3127**
### Description
A component to upload csv files into the application.
-Error handling is included
-Currently no backend to upload data into would update with axios for backend integration

### Testing Instructions

Open terminal and run "npm run dev"
Open application in browser with local host.
Import dropdown component and add to the template.

### Testing Instructions Storybook
Open terminal and run "npm run storybook"
Open storybook in browser "http://localhost:6006/".
Select theme you would like to view

### Screenshots
Successful Upload with file data on console

Error, No file selected

Storybook

### CODE REVIEW

File: frontend/src/components/Upload/Upload.vue
Comment (YoussefLaunchUp) [inline frontend/src/components/Upload/Upload.vue:1]: Should be renamed to index.vue

**Review by YoussefLaunchUp — CHANGES_REQUESTED** [review]
make sure to fix eslint issues as well

File: frontend/src/components/Upload/Upload.vue
Comment (kelvin3127) [reply frontend/src/components/Upload/Upload.vue:1]: Renamed and fixed eslint

**Review by YoussefLaunchUp — APPROVED** [review]
Good work

**Review by roxanehe — APPROVED** [review]
great work

**Review by neli-rashkova — APPROVED** [review]
Great work

**Pull request merged**